assets/training/model_management/src/azureml/model/mgmt/processors/transformers/convertors.py

```diff
@@ -246,6 +246,7 @@ class VisionMLflowConvertor(HFMLFLowConvertor):
 
     VISION_DIR = Path(__file__).parent / "vision"
     PREDICT_FILE_PATH = VISION_DIR / HFMLFLowConvertor.PREDICT_FILE_NAME
+    VISION_UTILS_FILE_PATH = Path(__file__).parent.parent / "common" / "vision_utils.py"
```

Comment:
nit: os join can be used.

Reply:
Thanks for the suggestion. Will address in a separate PR.